.travis.yml

```diff
@@ -35,8 +36,8 @@ before_install:
 before_script:
 - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl
 - sudo chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-- curl -Lo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64
-- sudo chmod +x minikube && sudo mv minikube /usr/local/bin/
+- if [[ "$MINIKUBE_VERSION" ]]; then curl -Lo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64; fi
+- if [[ "$MINIKUBE_VERSION" ]]; then sudo chmod +x minikube && sudo mv minikube /usr/local/bin/; fi
```

Comment:
Maybe sth like this here:
```
if [[ "$MINIKUBE_VERSION" ]]; then
    curl -Lo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64
    sudo chmod +x minikube && sudo mv minikube /usr/local/bin/
fi
```
It will be more compact